<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAsistencia</class>
 <widget class="QWidget" name="FormAsistencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de Asistencia</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f5f6fa;
    font-family: 'Segoe UI';
}
QLabel {
    color: #2c3e50;
    font-size: 14px;
}
QListWidget {
    background-color: white;
    border: 1px solid #dcdde1;
    border-radius: 4px;
    padding: 5px;
}
QPushButton {
    background-color: #e67e22;
    color: white;
    padding: 10px 20px;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #d35400;
}
QRadioButton {
    spacing: 10px;
    font-size: 13px;
}
QCalendarWidget QWidget {
    alternate-background-color: #ecf0f1;
}
QCalendarWidget QToolButton {
    color: black;
    icon-size: 20px, 20px;
    background-color: #ecf0f1;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Control de Asistencia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCalendar">
       <property name="text">
        <string>Seleccione Fecha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelEstado">
       <property name="text">
        <string>Estado:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupEstado">
       <property name="styleSheet">
        <string notr="true">QGroupBox { border: none; }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbPresente">
          <property name="text">
           <string>Presente</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbTardanza">
          <property name="text">
           <string>Tardanza</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbAusente">
          <property name="text">
           <string>Ausente</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMarcarAsistencia">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Guardar Estado</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLista">
       <property name="text">
        <string>Lista de Estudiantes (Seleccione uno):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listEstudiantes">
       <item>
        <property name="text">
         <string>001 - Ana García</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>002 - Luis Pérez</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>003 - María Rodriguez</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>004 - Carlos Sanchez</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>005 - Sofia Lopez</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
